### DISCUSSION TITLE
Trigger mode : Seq Play

### DISCUSSION THREAD

**Discussion by bukru** [bangcorrupt/hacktribe#312] [Q&A] [answered]
Hi all, i tried to use TRIGGER MODE = Seq Play in globals parameters.

It does not play anything when playing and trigger some part. Does anybody as the same ?

I tried while pattern is playing and all part muted. Normally it should play the pressed part while it's pressed or trigger the song one shot if quickly pressed.

See you

**Comment by bangcorrupt** [comment, marked as answer]
> I tried while pattern is playing and all part muted. Normally it should play the pressed part while it's pressed or trigger the song one shot if quickly pressed.

I don't think it's ever worked like that.  

Un-mute all the parts and stop the sequencer.  Then hold a pad and it will play the sequence for that part.

**Comment by bukru** [comment]
Haaaan ok... But it is stupid that it need to unmute part so you can't play with it while playing pattern ... And if you press play then you have all parts starting.... Korg ^^

Thanks again for précision
